app/static/main.css

```diff
@@ -117,4 +117,22 @@ h1{
     border-radius: 3px;
     margin-bottom: 20px;
     margin-right: 30px;
+  }
+
+  .profile-side-info {
+    background: #ffffff;
+    padding: 10px 20px;
+    border: 1px solid #dddddd;
+    border-radius: 3px;
+    margin-bottom: 20px;
+    margin-top: 30px;
+  }
+
+  .important-info-card {
+    background: #ffffff;
+    padding: 10px 20px;
+    border: 1px solid #dddddd;
+    border-radius: 3px;
+    margin-bottom: 20px;
+    margin-top: 30px;
```

Comment:
Just a quick copy/paste job to handle, what I'm calling, more "granular" styling in components down the tree.